packages/block-editor/src/components/block-list/index.js

```diff
@@ -276,7 +277,10 @@ export default compose( [
 			selectedBlockClientId: getSelectedBlockClientId(),
 			multiSelectedBlockClientIds: getMultiSelectedBlockClientIds(),
 			hasMultiSelection: hasMultiSelection(),
-			enableAnimation: getGlobalBlockCount() <= BLOCK_ANIMATION_THRESHOLD,
+			enableAnimation: (
+				! isTyping() &&
```

Comment:
I personally like seeing the animation while hitten "Enter" or using the slash inserter. Let's have more opinions here. @jasmussen @kjellr @mapk @mtias

Reply:
I personally don't like it on `Enter` :D It really distracts to writing and you kind of expect it to be instant.